dimod/generators/satisfiability.py

```diff
@@ -59,25 +67,39 @@ def random_kmcsat(variables: typing.Union[int, typing.Sequence[dimod.typing.Vari
     that consists in satisfying a number of conjunctive
     clauses of ``k`` literals (variables, or their negations).
     Each clause should encode a max-cut problem over the clause literals.
+    
+    Each clause contributes -:code:`k//2' to the energy when the clause is 
+    satisfied, and at least 0 when unsatisfied. The energy :math:`H(s)` for a 
+    spin assignment :math:`s` is thus lower bounded by :math:`E_{SAT}=-`:code:`k//2 * num_clauses`, 
```

Comment:
and single `` ` `` for math (unless you're doing special characters like `\sum`)